distribution/docs/src/test/java/ddf/docs/DocumentationTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.docs;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+public class DocumentationTest {
+
+    String errorMsg = "Unresolved directive";
+
+    String TARGET_DIRECTORY = "target";
+
+    String DOCS_DIRECTORY = "docs";
+
+    String HTML_DIRECTORY = "html";
+
+    @Test
+    public void testDocumentationIncludes() throws IOException, URISyntaxException {
+
+        Path htmlSubDirectory = Paths.get(System.getProperty("user.dir"), TARGET_DIRECTORY, DOCS_DIRECTORY, HTML_DIRECTORY);
```

Comment:
Not sure if that's going to do what we want. The only reason I suggested creating this variable was just to avoid calling `resolve()` multiple times on the next line. So if
```
Path htmlSubDirectory = Paths.get(TARGET_DIRECTORY, DOCS_DIRECTORY, HTML_DIRECTORY)
```
doesn't work, then I'd just get rid of that line and do
```
Path docPath = Paths.get(...).resolve(TARGET_DIRECTORY).resolve(DOCS_DIRECTORY).resolve(HTML_DIRECTORY);
```
on the next line.